src/SprykerEco/Zed/Unzer/UnzerConstants.php

```diff
@@ -248,4 +263,29 @@ interface UnzerConstants
      * @var string
      */
     public const WEBHOOK_EVENT_TYPE = 'all';
+
+    /**
+     * @var string
+     */
+    public const UNZER_BASKET_TYPE_GOODS = 'goods';
+
+    /**
+     * @var string
+     */
+    public const UNZER_BASKET_TYPE_SHIPMENTS = 'shipments';
+
+    /**
+     * @var string
+     */
+    public const UNZER_BASKET_SHIPMENT_TITLE = 'Shipping costs';
+
+    /**
+     * @var string
+     */
+    public const UNZER_BASKET_SHIPMENT_REFERENCE_ID = 'shipment_costs';
+
+    /**
+     * @var string
+     */
+    public const UNZER_MARKETPLACE_BASKET_SHIPMENT_REFERENCE_ID = self::UNZER_BASKET_SHIPMENT_REFERENCE_ID . '_%s';
```

Comment:
it probably worth to have method for this case
better for extendability

Reply:
There is no sense for extending cause it is linked to Unzer API call. I have renamed it for better readability.